### ISSUE TITLE
Cannot setup file blocked (Shared)

### ISSUE THREAD

**Issue by klabarge** [qzind/tray#430]
In QZ Tray 2.1, the below error is logged (several times) for each call to the API. It seems benign, but should be removed from the logs.

```
[DEBUG] 2019-03-04 20:46:53,861 @ qz.ws.PrintSocketClient:?
	Message: {"call":"printers.find","promise":{},"params":{"query":"zebra"},"timestamp":1551750413859,"uid":"0w87y9","position":{"x":960,"y":511.5}}
[WARN] 2019-03-04 20:46:53,863 @ qz.ws.PrintSocketClient:?
	Bad signature on request
[ERROR] 2019-03-04 20:46:53,866 @ qz.utils.FileUtilities:?
	Cannot setup file blocked (Shared)
java.io.IOException: No such file or directory
	at java.io.UnixFileSystem.createFileExclusively(Native Method)
	at java.io.File.createNewFile(File.java:1012)
	at qz.utils.FileUtilities.getFile(Unknown Source)
	at qz.auth.Certificate.isBlocked(Unknown Source)
	at qz.ws.PrintSocketClient.allowedFromDialog(Unknown Source)
	at qz.ws.PrintSocketClient.processMessage(Unknown Source)
	at qz.ws.PrintSocketClient.onMessage(Unknown Source)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.eclipse.jetty.websocket.common.events.annotated.CallableMethod.call(CallableMethod.java:70)
	at org.eclipse.jetty.websocket.common.events.annotated.OptionalSessionCallableMethod.call(OptionalSessionCallableMethod.java:68)
	at org.eclipse.jetty.websocket.common.events.JettyAnnotatedEventDriver$2.run(JettyAnnotatedEventDriver.java:210)
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:635)
	at org.eclipse.jetty.util.thread.QueuedThreadPool$3.run(QueuedThreadPool.java:555)
	at java.lang.Thread.run(Thread.java:748)
```

**Comment by tresf** [comment]
The "Shared" file code makes some assumptions and in certain circumstances (such as `allowed.dat` and `blocked.dat`) it's perfectly normal for QZ Tray to NOT be able to create these.

In other instances, (such as [`file-io`](https://qz.io/wiki/2.1-file-communication)), it's critical that the files can be created.

I'll keep the stacktrace (but with warning) on user (local) files and simply show a single line warning for shared files.

**Comment by tresf** [comment]
Closed via f83eb29.
